### ISSUE TITLE
Dashboard edit links not programmatically associated with what they edit

### ISSUE THREAD

**Issue by kimisgold** [omeka/Omeka#1117]
Description: On the 'Admin Dashboard' page, there are 'edit' links for all the items and collections lists. When using a screen reader and navigating through these components, all the edit links fail to have a programmatic label or other programmatic association with the item in the same row that it's going to edit.

Expected Results: Associated elements that are visually clear also have an appropriate programmatic association such as a table, list or proper ARIA structure.

Actual Results: Each item and collection has an edit link that isn't programmatically associated to the item or collection it is editing.

(1.3.1 Info and Relationships; Level A; WCAG 2.0)

**Comment by kimisgold** [comment]
Fixes are ready for a look on [`a11y/dashboard-grouping`](https://github.com/omeka/Omeka/tree/a11y/dashboard-grouping).

**Comment by aviannamiller** [comment]
I'm getting great results across browsers for VoiceOver and JAWS. For NVDA, all browsers, I am not picking up on any associated collection names on the Edit links, just for the "Recent Collections" section (so it's just the announcement of "Edit"). Item edit links read out appropriately.

**Comment by kimisgold** [comment]
I had missed actually assigning the group role to the Recent Collections container. Funny that Voiceover and JAWS still picked up on the aria labeling, but things should be fixed for NVDA now.

**Comment by aviannamiller** [comment]
Great, that works beautifully across screen readers!
